packages/framework/fluid-static/src/serviceAudience.ts

```diff
@@ -153,4 +153,11 @@ export abstract class ServiceAudience<M extends IMember = IMember>
     // Include only human members
     return member.details.capabilities.interactive;
   }
+
+    /**
+     * {@inheritDoc IServiceAudience._getInternalAudience}
+     */
+    public _getInternalAudience(): IAudience {
```

Comment:
why do you need to get this. its already on the IContainer.

Reply:
Ah, I missed that it was available through that route. Will update.

Reply:
Fixed on the dev branch